lib/executor.js

```diff
@@ -175,6 +175,7 @@ function setupDataSources(app, instructions) {
       useEnvVars: true,
     };
     obj = getUpdatedConfigObject(app, obj, opts);
+    obj.lazyConnect = process.env.LB_LAZYCONNECT_DATASOURCES;
```

Comment:
When the env var is not set, this ads `lazyConnect` variable with `undefined` value. I think this may surprise users when debugging. Also if `lazyConnect` was set in `datasources.json`, then you will re-set it back to whatever is provided by the env variable.

I am proposing a safer approach: 1) change the value only when the ENV var is set and also 2) handle "false" and "0" values as false

```js
var lazyConnect = process.env.LB_LAZYCONNECT_DATASOURCES;
if (lazyConnect) {
  obj.lazyConnect = lazyConnect === 'false' || lazyConnect === '0' ? false : true;
}
```